Common/src/adt/CADTElemClass.cpp

```diff
@@ -583,11 +585,11 @@ void CADTElemClass::Dist2ToElement(const unsigned long elemID, const su2double*
             function returns true. If the projection is not inside the triangle,
             false is returned and the distance to each of the lines of the
             triangle is computed and the minimum is taken. ---*/
-      su2double r, s;
+      su2double r = NAN, s = NAN;
```

Comment:
Don't initialize anything to NAN please.

Reply:
This comes from `cppcoreguidelines-init-variables` check/fixes. 

It is not adjustable I will change them manually. Leaving them uninitialized just leads to random numbers. What is a better choice `0.0`? 

In my view, `NaN` is not problematic, even helpful. At any point of code, we are not expecting to use created and untouched float variables initializing them to `NaN` also makes debugging easier. Where the error occurs the variable would be `NaN` if one trace back the variable it will be always `NaN` since the declaration, means something wrong with the function those variable used.

Reply:
It's ugly and not very useful because it is a quiet NaN, i.e. it won't be caught by the FP environment. So 0 is indeed better because at least you can catch the FP exception if you divide something.
What we should avoid is declaring variables without initializing them in the first place (which we've been requesting on new PRs). And folks are following it nicely and making things const in the process :) Is there a check for "variable could be const?"

Reply:
Unfortunately, there is no check for if a variable can be const.

There are checks about whether arguments can be const or not, whether const ref is better etc. 

I want to later add an [action](https://github.com/platisd/clang-tidy-pr-comments) which I think is quite nice for whom don't use clang-tidy locally. So I will change `NaN` to `0.0` rather than removing them.